tests/TestHarness/launch_transaction_generators.py

```diff
@@ -123,7 +123,7 @@ def launch(self, waitToComplete=True):
                         '--target-tps', f'{targetTps}',
                         '--log-dir', f'{self.logDir}',
                         '--peer-endpoint', f'{self.peerEndpoint}',
-                        '--port', f'{self.port}'
+                        '--port', f'{self.port}',
```

Comment:
Why?

Reply:
Lingering from the merge where I previously added a key to pass along. Removed now https://github.com/AntelopeIO/leap/pull/713/commits/724ba6cdae9cf9c3f7a7a1c8a92c88e468b4da3a

Reply:
If you remove the added space your PR will not have touched `launch_transaction_generators.py`.  Not sure it is worth adding a space when you could have a clean diff here and the new space isn't really doing anything.